gen/gen_sim.py.j2

```diff
@@ -42,3 +42,14 @@ class {{ module_name }}{% if inherits is defined %}({{ inherits }}){% endif %}:
 
   {% endif %}
   {% endfor %}
+
+  {% if module_name == 'MotController' %}
+    def _is_follower(self, id):
+        from ctre._impl.autogen.ctre_sim_enums import ControlMode
```

Comment:
Add this import to `sim_extra`? And it should be `from .ctre_sim_enums import ControlMode`.